src/analysis/retail/warlock/demonology/CONFIG.tsx

```diff
@@ -7,10 +7,10 @@ import CHANGELOG from './CHANGELOG';
 
 const config: Config = {
   // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
-  contributors: [Zyer, Gazh],
+  contributors: [Zyer, Gazh, Leftyxiv],
   branch: GameBranch.Retail,
   // The WoW client patch this spec was last updated.
-  patchCompatibility: '11.0.2',
+  patchCompatibility: '11.1.5',
```

Comment:
If something as "basic" resource handling was off - is it really compatible with the current patch with no other changes? It might be, I don't know, just seems odd if something like that was the only thing 'off'